src/integrationTest/java/com/splunk/ibm/mq/integration/tests/WMQMonitorIntegrationTest.java

```diff
@@ -96,6 +147,13 @@ void test_monitor_with_full_config() throws Exception {
 
     TestWMQMonitor monitor = new TestWMQMonitor(configFile, metricWriteHelper);
     monitor.testrun();
+    List<MetricData> data = testExporter.getExportedMetrics();
+    Set<String> metricNames = new HashSet<>();
+    for (MetricData metricData : data) {
+      metricNames.add(metricData.getName());
+    }
+    // this value is read from the configuration queue.
+    assertThat(metricNames).contains("mq.manager.max.handles");
```

Comment:
oh, i see,  so so far just one assertion.  we will add more?

Reply:
yes definitely more to come